integration/nwo/deploy.go

```diff
@@ -46,6 +46,11 @@ type Chaincode struct {
 	Label               string
 	SignaturePolicy     string
 	ChannelConfigPolicy string
+	Server              bool
+}
+
+func (c *Chaincode) ID() string {
+	return c.Name
```

Comment:
This accessor should not be needed. afaict, the only call is when you're reserving ports and that should be part of the test.